plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/DeletionVectorWriter.java

```diff
@@ -132,16 +135,15 @@ public List<String> rewrittenDeleteFiles()
     @Override
     public FileMetrics getFileMetrics()
     {
-        // TODO Fix metrics
-        DeleteFile deleteFile = result.deleteFiles().getLast();
+        DeleteFile deleteFile = deleteFile();
         Metrics metrics = new Metrics(deleteFile.recordCount(), deleteFile.columnSizes(), deleteFile.valueCounts(), deleteFile.nullValueCounts(), deleteFile.nanValueCounts());
```

Comment:
add two more params to Metrics -> , deleteFile.lowerBounds(), deleteFile.upperBounds()?